compiler/pash_compiler.py

```diff
@@ -92,9 +93,15 @@ def compile_ir(ir_filename, compiled_script_file, args, compiler_config):
         ret = compile_optimize_output_script(
             ir_filename, compiled_script_file, args, compiler_config
         )
+    except UnparallelizableError as e: 
+        log("WARNING: Exception caught because some region(s) are unparallelizable:", e) 
+        # log(traceback.format_exc()) # uncomment for exact trace report (PaSh user should see informative messages for unparellizable regions) 
+    except (AdjLineNotImplementedError, NotImplementedError) as e: 
+        log("WARNING: Exception caught because some part is not implemented:", e)
+        log(traceback.format_exc())
     except Exception as e:
         log("WARNING: Exception caught:", e)
-        # traceback.print_exc()
+        log(traceback.format_exc())
```

Comment:
Konstantinos (from old branch): Since this will now happen less often it might be OK to let the traceback be logged.